movies/src/components/movieList/MovieList.js

```diff
@@ -29,34 +30,43 @@ const MoviesList = (props) => {
     <>
       { isMovieListLoaded ?
         <div className="movie-list">
-          <ResultCount
-            count={movies.length}
-          />
-          { movies.length ? movies.map((movie) => (
-            <MovieCard
-              title={movie.title}
-              genre={setMovieGenre(movie)}
-              year={setMovieYear(movie)}
-              releaseDate={movie.release_date}
-              src={movie.poster_path}
-              url={movie.poster_path}
-              id={movie.id}
-              key={movie.id}
-              overview={movie.overview}
-              runtime={movie.runtime}
-              selectMovie={selectMovie}
+          <Route path="/search/*">
+            <ResultCount
+              count={movies.length}
             />
-          ))
-          :
-            null
+          </Route>
+          {
+            movies.length ? movies.map((movie) => (
+              <Route path={["/search/*", "/film"]}>
```

Comment:
key prop is missing

Reply:
Should I add key to  Route or adding key for MovieCard is enough ?